src/operator/quantization/quantize_graph_pass.cc

```diff
@@ -130,6 +131,7 @@ Graph QuantizeGraph(Graph &&src) {
   // graph. Key is the currently visited graph's node pointer, and value is a copied node of the key
   // node. The existing key's value may be updated with the newly created quantize/dequantize op.
   std::unordered_map<Node*, NodePtr> mirror_map;
+  nnvm::NodeEntryMap<NodeEntry> multiple_outputs_entry_map;
```

Comment:
I would prefer a generic way to handle both single and multiple outputs than treating them differently

Reply:
Good point here.
This patch is trying to keep the mapping relation as before with minimal change. Actually it's able to handle both single and multiple outputs via their entry instead of current implementation, and thus no need to update the mirror_map's value to quantize/requantize op.
I will update the code to make it uniform for single and multiple outputs cases.